db/models.py

```diff
@@ -14,3 +16,37 @@
 
     def __str__(self) -> str:
         return f"{self.first_name} {self.last_name}"
+
+
+class Movie(models.Model):
+    title = models.CharField(max_length=255)
+    description = models.TextField()
+    actors = models.ManyToManyField(Actor, related_name="actors")
+    genres = models.ManyToManyField(Genre, related_name="genres")
```

Comment:
The `related_name` for the `genres` field in the `Movie` model is set to 'genres', which is the same as the field name. It's recommended to use a different `related_name`, such as 'movies' or 'movie_genres', to avoid confusion and potential conflicts.